### PR TITLE
chore: [MAM-283] Set up capistrano deployment via github action

### PR DESCRIPTION
**Pull request by felipecsl**

### CODE REVIEW

File: config/deploy_id_rsa_enc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+U2FsdGVkX1+qgmt3SzdcKx/FVhDrovWbFoetH5KIvym6073I2J0fhByPAY2svETO
```
Comment (sdubinsky) [inline config/deploy_id_rsa_enc:1]: are you sure this should be in a commit?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -63,3 +63,19 @@ jobs:
         run: |
           bin/rails assets:precompile
           bin/rails db:create db:schema:load db:seed spec
+  deploy:
+    runs-on: ubuntu-latest
+    needs: test
```
Comment (sdubinsky) [inline .github/workflows/tests.yml:68]: if I'm reading this right, it means if the test step fails it won't deploy?  We should do the same for the lint step.

File: config/deploy_id_rsa_enc (hunk 1 above)
Comment (felipecsl) [reply config/deploy_id_rsa_enc:1]: yup, this is the encrypted key that capistrano-deploy will use to connect to the server and pull the code from github

File: .github/workflows/tests.yml (hunk 2 above)
Comment (felipecsl) [reply .github/workflows/tests.yml:68]: that's right! the lint step is in a different workflow so I don't think we can refer to that easily. if we move linting here (we could) then yes we can do that

Comment (sdubinsky) [reply .github/workflows/tests.yml:68]: something for the future, then

**Review by sdubinsky — APPROVED** [review]

**Pull request merged**